lib/connectors/mongodb/mongo_rules_parser.rb

```diff
@@ -16,24 +16,22 @@ class MongoRulesParser < Connectors::Base::SimpleRulesParser
       def parse_rule(rule)
         field = rule.field
         value = rule.value
-        unless value.present?
-          raise "value is required for field: #{field}"
-        end
-        unless field.present?
-          raise "field is required for rule: #{rule}"
-        end
         op = rule.rule
         case op
-        when Core::Filtering::SimpleRule::Rule::EQUALS
+        when SimpleRule::Rule::EQUALS
```

Comment:
How did you shorten these? I thought you could only omit the fully-qualified prefix if these were in the same module?

Reply:
I did `include Core::Filtering` in the class. =) (See row 18 above for the `lib/connectors/base/simple_rules_parser.rb`).